authors/settings.py

```diff
@@ -37,7 +37,7 @@
     'django.contrib.staticfiles',
 
     'corsheaders',
-    'django_extensions',
+    
```

Comment:
Hey @larryTheGeek, I'm curious about this. Why did you choose to remove django extensions?

Reply:
It was a mistake. Let me rectify it immediately